frontend/app-development/layout/version-control/VersionControlHeader.tsx

```diff
@@ -157,22 +160,27 @@ export const VersionControlHeader = (props: IVersionControlHeaderProps) => {
       });
     }
   };
+
   const repoPushMutation = useRepoPushMutation(org, app);
   const pushChanges = async () => {
+    setAltinnSpinnerState(true);
     setModalState({
       ...initialModalState,
       header: t('sync_header.sharing_changes'),
       isLoading: true,
     });
-    await repoPushMutation.mutate();
+    repoPushMutation.mutate();
     setHasChangesInMaster(false);
     setHasChangesInLocalRepo(false);
```

Comment:
These two lines (168 and 169) can be removed, as the mutation query actually triggers these being set correctly when it re-loads the data

Reply:
Removed